### ISSUE TITLE
Do not increase early food crops needed, instead increase only later ones

### ISSUE THREAD

**Issue by Azzurite** [D3nnis3n/White-Tiger-Feedback#15]
So on the current cycle 25, it has been pretty clear that the Campfire Cooking crop requirements are massive. The first 1-2 days, everyone was gathering, trying to sell to cooks, taking up loans to pay for food etc. This also resulted in a massive amount of players going Gathering, of course many also just because the change had been advertised.

However, what has always been a problem in Eco (and in this way simulates real life rather well) is that early on, you need a lot more people gathering/farming crops than in the later stages of the game. On vanilla servers, you basically don't see much farming until hand plows hit. Then, once tractors get introduced, the amount of farmers you need to supply the whole server probably gets cut in half or even cut to a third.

However, what happened in real life might not be good for the game itself. The development of "basically everyone needs to be farmer" -> "only a few people need to be farmers" happened in RL over hundreds of years. Farmers died and less new people became farmers. But in Eco, you do this whole development within a server cycle. "Old" farmers do not die out, they're still farmers. So if we need 40 farmers at the beginning of the cycle, only 25 when hand ploughs hit, and 10 when steam tractors come in, what do the rest of the people who dedicated their cycle to farming at the beginning do? This is not viable game design, at least for these players.

This situation is now even more pronounced as the crop amounts that are needed for charred + campfire food have been greatly increased compared to vanilla levels.

There are a few ideas I can currently come up with to fix this.
<ol>
<li>First is that there was an invention much earlier than the hand plough, that allowed higher efficiency: the ard, a plough pulled by an animal. In fact, domestication of animals themselves happened even earlier and could also remedy the early food situation. However, obviously, none of these things are in game, so this is not really a solution.</li>

<li>Secondly is allowing skills in the gatherer skill tree to be re-specialized into something else once the need for farmers gets less. This obviously goes against the spirit of the game and I can't think of how to do this fairly and without it feeling bad.</li>

<li>Thirdly, I think the whole food rebalancing might be thought about backwards. The goal is to have farming viable into the later parts of the game and to need a lot of the world be farmland. I don't think you achieve that by touching early foods at all! Early foods are created by people gathering over the world. Farming is not viable at all since crops take 20 hours to grow, Farming is not researched so there are no seeds anyway, and fields have to be painfully created with hoes.
  
Increasing the crops needed for early foods does exactly nothing to need more farmland. In fact, the only thing that it does is make Gathering less profitable per each Gatherer! Since people still need the same amount of food, but now suddenly Gatherers have to work a lot more to supply that food, so there need to be more Gatherers. While previously, idk, 10% of the people on the server needed to be Gatherers, now it's closer to 30%.

A second thing that happened is that freely gathered food actually started to run low or disappear. The map just simply doesn't support the values needed for everyone's foods.

So, I think the crop amounts for "Campfire Cooking" food should've just stayed the same as in the default game. Charred, non-"Campfire Cooking" foods could have still had increased values, but maybe not quite as much. 
That way, early on, you only need the "normal" amount of Gatherers and they'll be able to fill all the food needs. No extreme devaluation of crops -> less Gatherers needed -> Gatherers earn more.
Then, once people get their second point, these gatherers go into farming. Hopefully by that point, hand plows aren't too far off and they can start to build bigger farms immediately, starting with all the seeds that were gathered from wild plants all the while before.

A little bit later, Cooking/Milling/Baking should hit and NOW is when a lot more crops should start to be required for food. The new foods should need more crops like the early foods do now. NOW, the big farms start to be really necessary to get quality food, which also matches the farmer gameplay progression where they also start to produce more crops.

This should be even more pronounced when the Advanced Cooking/Baking skills hit - these foods should require even more crops. At this point there should be tractors, so the amounts farmed and the amount of crops needed balance each other out again.

If done this way, it is ensured that early on, you don't need an excessive amount of gatherers/farmers, and their number stays the same throughout the cycle, i.e. people that were Gatherers/Farmers at the beginning stay useful and needed through the later part of the game, and still reaching the end goal of requiring a lot of land to be farmland.
</li></ol>

Now, I think this third solution is the correct one, but for this third solution to work, there need to be a few more changes.

The immediate problem I'd see with it, is that people just eat lesser quality food and deal with getting a bit less XP. They'd progress slower in their skills, but since the later foods require so much more crops, and crop production because of hand ploughs/tractors increases greatly, these less quality food will simply be used in mass as they get incredibly cheap. This already happens on vanilla servers without any modded increase in crops for higher quality foods.

The first idea to rectify that would be to make XP multipliers scale to a power of X with better food. Currently (vanilla, dunno the WT values), you get ~`x60` XP from Campfire Cooking, ~`x90` from Cooking/Baking and ~`x120` XP from Advanced Cooking/Baking. This could be changed (for example) to `x60` XP from Campfire Cooking, `x120` XP from Cooking/Baking and `x240` XP from Advanced Cooking/Baking. If you then also increase the XP values necessary for the endgame skills like electronics dramatically, it would basically require you to eat better food to progress fast enough in these endgame skills at all.
Of course, if you do that, you have to do the same with housing items, otherwise they become useless. This would be hard to get right as well.
In my opinion, this approach is hard, as it requires very careful balancing to get right, and messes with the balance between food-housing as well.

Another approach would be to introduce a tier system for food: if you eat "Tier 1" (Campfire Cooking) food, their XP multiplier applies to "Basic Engineering" but the XP multiplier doesn't apply to "Mechanics". "Tier 2" (Baking/Cooking) food would apply their XP multiplier to "Basic Engineering" and "mechanics", but not to "Electronics". "Tier 3" (Advanced Cooking/Baking) would apply their XP multiplier to all skills. That way, you require the use of better food and you can also require the use of more crops resulting in more farm land. 
You could also scale this differently than "all-or-nothing", i.e. "Tier 1" food applies its full XP multiplier to "Basic Engineering", 0.3 times the multiplier to "Mechanics" and 0 to "Electronics", "Tier 2" food applies 0.8 times to "Basic Engineering", fully to "Mechanics", and 0.3 times to "Electronics", "Tier 3" food could apply 0.5 times to "Basic Engineering", 0.8 times to "Mechanics" and fully to "Electronics".

Personally, I think this tiered system of food is a really good way to do food in general, not only on White Tiger. I think if implemented correctly with the right values, it would get rid of the imbalance of "many gatherers/farmers early" - "little amount of farmers/gatherers late", which is a big problem for farmers, while also making farmers & chefs important later into the game.

So to summarize: Reduce crops needed for earlier foods, and increase crops needed for later foods. Then implement a system to actually require eating the later foods to progress through other skills, ensuring that later foods are actually eaten & required.

**Comment by Azzurite** [comment]
From Discord:

> However, I did not think about what happens after leveling to lvl 7, which is also one of the main problems. But I think this can be solved the same way: have calories also be divided into tiers. Higher tier calories apply 100% of the calories when used for an Electronics recipe, while tier 0 charred food only applies 10% of their calories to a project.

**Comment by Azzurite** [comment]
Here is a mod that implements a similar thing, but instead of increasing amount needed or XP decrease, it decreases the crafting speed: https://eco.mod.io/more-important-food

It also only acts on XP gained from food, which honestly is not a bad measurement to use instead of "tiers", because they're directly related.
